assets/stylesheets/kitten/components/headers/_header.scss

```diff
@@ -119,6 +125,14 @@
     display: flex;
   }
 
+  .k-Header__logo--smallOnXxs {
+    svg, img {
+      @include k-media-max('xxs') {
+        width: 120px;
```

Comment:
Sur Kitten, on essaye d'utiliser l'unité `rem`. Il y a une mixin pour faire la conversion en partant des pixels => `k-px-to-rem(120px)`

Reply:
Merci, je mets à jour!